hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieActiveTimeline.java

```diff
@@ -113,7 +113,9 @@ protected HoodieActiveTimeline(HoodieTableMetaClient metaClient, Set<String> inc
     // multiple casts will make this lambda serializable -
     // http://docs.oracle.com/javase/specs/jls/se8/html/jls-15.html#jls-15.16
     this.details = (Function<HoodieInstant, Option<byte[]>> & Serializable) this::getInstantDetails;
-    LOG.info("Loaded instants " + getInstants().collect(Collectors.toList()));
+    if (LOG.isDebugEnabled()) {
```

Comment:
This is actually a helpful debug aid, to tell us what the state of the timeline was. So I am changing this to print the last instant , while moving it back to INFO